src/libstd/sys/vxworks/process/process_vxworks.rs

```diff
@@ -1,10 +1,13 @@
+// Copyright (c) 2019 Wind River Systems, Inc.
```

Comment:
License?

Reply:
@lzutao 
Thanks for review on this.
Yes, we would like to add the License text for every file under vxworks directory. Is this a problem?

Reply:
The intent is "the usual Rust license", i.e. Wind River is not claiming any proprietary rights here. But I'm not a lawyer, let me check with someone who can give a better answer and get back to you.